### PR TITLE
Update cabot role to use pip and systemd

### PR DESCRIPTION
**Pull request by frankh**

### CODE REVIEW

**Comment by frankh** [comment]
I think maybe we need a restart handler after installing cabot/updating systemd files?

File: tasks/main.yml [hunk 1]
```diff
@@ -9,25 +9,13 @@
     - python-dev
```
Comment (JeanFred) [inline tasks/main.yml:9]: Do we still need python-dev ?

File: tasks/main.yml [hunk 2]
```diff
@@ -37,25 +25,16 @@
...
-- name: Install PIP modules
+- name: Install Cabot from pip
   pip:
-    requirements: '{{ cabot_config.app_path }}/{{ item }}'
+    name: "cabot=={{ cabot.version }}"
     virtualenv: '{{cabot_config.venv_path}}'
     virtualenv_site_packages: no
-  with_items:
-    - requirements.txt
-    - requirements-plugins.txt
```
Comment (JeanFred) [inline tasks/main.yml:57]: How about the plugins?

Comment (frankh) [reply tasks/main.yml:57]: They'll be installed by pip (they are all part of the package requirements)

We need a system for custom plugins though

File: tasks/main.yml (hunk 1 above)
Comment (frankh) [reply tasks/main.yml:9]: Maybe? some of the requirements may need it to build

File: tasks/main.yml (hunk 2 above)
Comment (JeanFred) [reply tasks/main.yml:57]: I meant custom plugins yes :)

**Review by JeanFred — APPROVED** [review]

**Pull request merged**